### ISSUE TITLE
FR - Would it be possible to keep the highlight for the matched text "search-result-file-matched-text" in "render markdown" mode ?

### ISSUE THREAD

**Issue by nicolasmarc1** [nothingislost/obsidian-query-control#15]
Hi!
Thanks for this amazing plugin which allows us to finally have rendered search results and backlinks (which i hope get natively supported by obsidian one day). 
I just wanted to know if it would it be possible to keep the highlight for the matched text "search-result-file-matched-text" in the "render markdown" mode in order to easily spot the searched text/backlink in a large text (or an other way to do so) . 
If it's not easily doable it's okay as we already have the alternative to toggle between render and unrendered mode (to spot where are the occurrences of the searched text/backlink) and render mode to see it full rendered.
Thanks again!

**Comment by simonfossom** [comment]
Yeah, this is actually a bug since the original functionality is not preserved. 

+1 from me
